packages/shared/src/hooks/useUpdatePost.ts

```diff
@@ -16,6 +16,13 @@ export default function useUpdatePost(): UseBookmarkPostRet {
       const postQueryKey = ['post', props.id];
       await client.cancelQueries(postQueryKey);
       const oldPost = client.getQueryData<PostData>(postQueryKey);
+
+      if (!oldPost) {
+        // nothing in cache so we skip any update
+
+        return () => undefined;
```

Comment:
Would a `return;` not be enough here?

Reply:
We use this return value for rollback in `mutation.onError` so want to avoid and `undefined()` calls which would break parts of the app.

Reply:
Alright then